src/app/modules/item/pages/item-details/item-details.component.html

```diff
@@ -12,6 +12,59 @@
       (groupJoined)="reloadItem()"
     ></alg-access-code-view>
 
+    <ng-container *ngIf="watchedGroup$ | async as watchedGroup">
+      <alg-section-paragraph
+        [theme]="'warning'"
+        icon="fa fa-flag"
+        [collapsible]="true"
+        [collapsed]="true"
+      >
+        <ng-container label-section i18n>
+          «{{ watchedGroup.name }}»
+          <ng-container
+            *ngIf="['content', 'content_with_descendants', 'solution'].includes(itemData.item.permissions.canView); else cantBlock">
+            can
+          </ng-container>
+          <ng-template #cantBlock>can't</ng-template> access this
+          <ng-container *ngIf="itemData.item.type === 'Skill'; else activity">skill</ng-container>
+          <ng-template #activity>activity</ng-template>
+        </ng-container>
```

Comment:
@smadbe I'd like your opinion here, I have no idea how angular i18n with nested ngIf & ng-template works, will it work as expected for i18n ?

Reply:
No way it can work with ifs, containers, variables... the fact that "can't" can be prepended at this position works only in some languages. That is incorrect use of localization. Check the doc.

Reply:
Fixed